src/Client.php

```diff
@@ -161,29 +194,60 @@ private function post($route = '', array $body = [])
     }
 
     /**
-     * Perform cURL DELETE request to route. Used by
+     * Perform cURL GET request to route. For example used by
      * Alerts API requests: https://www.stathat.com/manual/alerts_api
      *
-     * @param string $route
-     * Ex: /x/ACCESSTOKEN/alerts/ALERTID
+     * @param string $curl_url
+     *   Ex: https://www.stathat.com/x/ACCESSTOKEN/alerts
+     * @return array or string $result
+     *   An array of alert details or the response code string in the case of a non 200 response.
      */
-    private function deleteAlertCurl($route = '')
+    private function curlGet($curl_url)
```

Comment:
Not necessary for this pull request, but all these extra methods are making me think we definitely need to move these out of this class. (#8) I also wonder if we could get away with having a cURL-based method for each HTTP verb (so just `get`, `delete`, ...), and then an `asyncPost` method that makes the (hopefully non-blocking) request without waiting for a response.